bank/src/main/java/org/example/Bank/Bank.java

```diff
@@ -0,0 +1,192 @@
+package org.example.Bank;
+
+
+//util pack import
+
+import org.example.Account.Account;
+import org.example.Account.savingAccount;
+import org.example.Exception.UnknownException;
+import org.example.Exception.WithdrawException;
+
+import java.math.BigDecimal;
+import java.text.DecimalFormat;
+import java.util.HashMap;
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
+
+//exception part
+
+/*
+ * 			계좌 클래스에서 구현한 기본 클래스를 이용하여 계좌생성, 출금, 입금, 송금 메서드를 구현합니다. 메서드 내부적으로 입력값을 받는 액션이 있습니다.
+ *
+			적금 계좌는 적금 계좌는 잔액이 목표 금액(%s원) 이상이어야 출금 가능하도록 상속받은 출금 메서드를 조금 다르게 구현해줍니다.
+
+ 모든 계좌를 관리하기 위한 중앙은행 클래스를 만듭니다. 계좌를 배열에 관리하기 위해 ArrayList를 활용합니다.
+계좌별 이자 정책을 관리하기 위해 HashMap을 활용하고, key에 category, value에 각 category의 InterestCalculator 인스턴스 를 담습니다.
+ */
+
+public class Bank {
+    protected static Scanner scanner = new Scanner(System.in);
+    protected static int seq = 0;
+    public static DecimalFormat df = new DecimalFormat("#,###");
+
+    ///이자계산을 위한 해시맵
+
+
+    private HashMap<String, InterestCalculator> interestCalculators = new HashMap<>();
```

Comment:
```suggestion
    private final HashMap<String, InterestCalculator> interestCalculators = new HashMap<>();
```
final키워드를 사용해 해당 인스턴스는 변하지 않을 것이라는 것을 명시해야 합니다